.rubocop.yml

```diff
@@ -8,7 +8,7 @@ AllCops:
 
 # We like longer lines
 Metrics/LineLength:
-  Max: 160
+  Max: 199
```

Comment:
You don't need to change this value anymore, I think you changed it for the test that is now removed ;)